test/codegen-generated-classes-test/src/test/java/software/amazon/awssdk/services/ExecutionAttributesTest.java

```diff
@@ -69,4 +67,131 @@ public void asyncClient_disableHostPrefixInjection_isPresent() {
             async.close();
         }
     }
-}
+
+    @Test
+    public void asyncClient_clientOverrideExecutionAttribute() {
```

Comment:
Please add tests for the sync clients as well

Reply:
Initial change was to do this only for async clients as the pointer you sent me is only called by async clients. Would you like me to implement for sync clients as well?

Reply:
Sorry about that. Yes, since the options are available through the sync paths, the sync clients should support it as well.